.github/workflows/ci.yml

```diff
@@ -158,7 +158,7 @@ jobs:
           key: macos
       - name: Build OpenRCT2
         run: |
-          brew install ninja
+          HOMEBREW_NO_ANALYTICS=1 brew install ninja
```

Comment:
Why? What does it bring to the table? Maybe add the reasoning to the commit body

Reply:
What about this one?